### PR TITLE
Tweak design system example for per-component packages

### PR DESCRIPTION
**Pull request by jaredpalmer**

### CODE REVIEW

File: examples/design-system/packages/eslint-preset-acme/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# `scripts`
```
Comment (weyert) [inline examples/design-system/packages/eslint-preset-acme/README.md:1]: Shouldn’t this be eslint-preset-acme instead?

**Comment by mr-bjerre** [comment]
Question copied from twitter

> Isn’t preferred to have a single ui package with multiple components (and then maybe add tree shaking)? Otherwise you end up with loads of packages and dependencies in package.json. Interested in the reasoning for this approach instead

**Comment by mr-bjerre** [comment]
Also sidenote; currently `yarn lint` is failing in design system example...

File: examples/design-system/packages/eslint-preset-acme/README.md (hunk 1 above)
Comment (jaredpalmer) [reply examples/design-system/packages/eslint-preset-acme/README.md:1]: Yes

**Comment by jaredpalmer** [comment]
Yeah, it really depends on how good tree shaking is—tree shaking is historically very finicky. I’m tempted to close this and leave the existing setup as is. If people want this per-component packages, they can easily move to it on their own.

**Comment by williamdespard** [comment]
Hi @jaredpalmer and @mr-bjerre, just to clarify, does tree shaking come included out of the box when importing from a single ui package in Turborepo? Or is there some extra setup required? If so, it would be great if you could point me in the direction of some docs.

**Pull request closed without merging**